src/utils/prepareTodos.ts

```diff
@@ -0,0 +1,24 @@
+import { Todo } from '../types/Todo';
+import { SortType } from '../types/SortType';
+
+export const prepareTodos = (todos: Todo[], sortType: SortType): Todo[] => {
+  let visibleTodos = [...todos];
+
+  visibleTodos = visibleTodos.filter(todo => {
+    switch (sortType) {
+      case SortType.ACTIVE:
+        return !todo.completed;
+
+      case SortType.COMPLETED:
+        return todo.completed;
+
+      case SortType.ALL:
+        return true;
+
+      default:
```

Comment:
you have typization for this, so you will never get unexpected type, let us move `All` to default case